maat/tests/real_world.rs

```diff
@@ -312,13 +311,31 @@ async fn real_world_use_case() {
 
     register_storage_provider(&client, &charlie_kp).await;
     // Set PoRep VerifyingKey extrinsic only accepts scale-encoded bytes of Verifying Key in substrate form.
-    let vk =
-        polka_storage_proofs::VerifyingKey::<bls12_381::Bls12>::try_from(parameters.vk).unwrap();
-    let vk_scale = Encode::encode(&vk);
-    set_porep_verifying_key(&client, &charlie_kp, VerifyingKey::from_raw_bytes(vk_scale)).await;
+    let porep_vk =
+        polka_storage_proofs::VerifyingKey::<bls12_381::Bls12>::try_from(porep_parameters.vk)
+            .unwrap();
+    let porep_vk_scale = Encode::encode(&porep_vk);
+    set_porep_verifying_key(
+        &client,
+        &charlie_kp,
+        VerifyingKey::from_raw_bytes(porep_vk_scale),
+    )
+    .await;
+
+    let post_vk =
+        polka_storage_proofs::VerifyingKey::<bls12_381::Bls12>::try_from(post_parameters.vk)
+            .unwrap();
+    let post_vk_scale = Encode::encode(&post_vk);
+    set_post_verifying_key(
+        &client,
+        &charlie_kp,
+        VerifyingKey::from_raw_bytes(post_vk_scale),
+    )
+    .await;
 
     // Add balance to Charlie
-    let balance = 12_500_000_000;
+    // collateral + pre_commit_deposit (1)
+    let balance = 12_500_000_001;
```

Comment:
Nit: you can expose the SP/Market function that defines the value and query it here
Or at least leave a note for one of us shmucks that will do that